### PR TITLE
add python quick start guide

### PR DESCRIPTION
**Pull request by yoni206**

### CODE REVIEW

File: docs/api/python/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+  unsat core: [(< 0 a), (< 0 b), (< (+ a b) 1)]
+
+Example
+-------
+
+| The SMT-LIB input for this example can be found at `examples/api/smtlib/quickstart.smt2 <https://github.com/cvc5/cvc5/blob/master/examples/api/smtlib/quickstart.smt2>`_.
+| The source code for this example can be found at `examples/api/python/quickstart.py <https://github.com/cvc5/cvc5/blob/master/examples/api/python/quickstart.cpp>`_.
+
+.. api-examples::
+    ../../../examples/api/python/quickstart.cpp
```
Comment (yoni206) [inline docs/api/python/quickstart.rst:172]: wrong path.

File: docs/api/python/quickstart.rst [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+We will ask the solver to produce models and unsat cores in the following,
+and for this we have to enable the following options.
+
+.. literalinclude:: ../../../examples/api/python/quickstart.py
+     :language: python
+     :lines: 26-27
+
+Next we set the logic.
+The simplest way to set a logic for the solver is to choose "ALL".
```
Comment (nafur) [inline docs/api/python/quickstart.rst:18]:
```suggestion
The simplest way to set a logic for the solver is to choose ``"ALL"``.
```

**Review by nafur — APPROVED** [review]
Just one minor thing, otherwise LGTM.

**Review by aniemetz — APPROVED** [review]
Looks great! Thanks for taking care of this!

**Pull request merged**